try:
    print(number)

except:
    print("An error ocurred")

finally:
    print("Success")

x = 67
y = 0

try:
    print(x / y)

except:
    print("Arithmetic error ocurred")

finally:
    print("Success")
